tinyagentos/scheduler/gpu_arbiter.py

```
@@ -0,0 +1,330 @@
+                return GpuAdmission(
+                    admitted=False, free_vram_mb=free_vram, required_vram_mb=required_vram_mb,
+                    reason=f"insufficient local VRAM: need {required_vram_mb} MiB, have {free_vram} MiB free",
+                )
+            return GpuAdmission(admitted=True, free_vram_mb=free_vram, required_vram_mb=required_vram_mb)
+        if self._cluster_manager is not None:
+            leases = self._cluster_manager.get_leases()
+            for worker in self._cluster_manager.get_workers():
+                if worker.status != "online":
+                    continue
+                worker_leases = sum(
+                    l.required_vram_mb for l in leases
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)